furusiya/model/skills/omnislash.py

```diff
@@ -0,0 +1,9 @@
+import random
+
+
+class OmniSlash:
+    @classmethod
+    def process(cls, player, rehit_percent, delta_coords):
```

Comment:
Instead of modelling it this way (static class method with all params), I would add `rehit_percent` and maybe `player` to the initializer and make this a class you can create instances of.  That's just my choice/preference though (can't argue for why).

Reply:
This would necessitate one of two things:
- Keeping an instance around with the player. Booo-riiing.
- Creating a new instance every time the player hits something. Also booo-riiing.

You see, keeping an instance around would break live `config.json` updating, and creating a new instance would defeat the entire point: we're still passing the things as we're calling the method.

Might as well pass everything into the method.